### PR TITLE
Allow config.yaml loading to support overriding siblingings url by re…

### PR DESCRIPTION
**Pull request by JC1738**
…flecting map[string]string from env variable

Also fixed some linting in messages.go

### CODE REVIEW

**Comment by sous-machine-account** [comment]
LGTM

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: util/configloader/configloader_test.go [hunk 1]
```diff
@@ -89,3 +93,24 @@ func TestLoad_EmptyEnv(t *testing.T) {
...
+	if err := cl.Load(&c, "test_map_config.yaml"); err != nil {
+		t.Fatal(err)
+	}
+
+	if _, ok := c.SiblingURLs["env3"]; ok {
+		t.Errorf("Failed to override map from environment variable")
+	}
+
+	if val := c.SiblingURLs["env1"]; val != "foo" {
+		t.Errorf("Value not expected")
```
Comment (samsalisbury) [inline util/configloader/configloader_test.go]: In general I always print out the actual and expected values in the error message as well, as it's often helpful in quickly identifying the cause (e.g. typos). But this case is pretty simple so I can live with this.

File: util/configloader/test_map_config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+SiblingURLs:
+  env1: http://foo.com
+  env2: http://bar.com
+  env3: http://example.com
```
Comment (samsalisbury) [inline util/configloader/test_map_config.yaml]: In general it's a good idea to put data files like this in a `testdata` sub directory, as this is treated specially by the go toolchain and will be explicitly ignored when compiling etc. For one file it makes little difference, but prob worth doing anyway as it's good practice. (See `go help packages`)

**Comment by JC1738** [comment]
@samsalisbury Made changes suggested by review and ready for re-review/merge

**Comment by samsalisbury** [comment]
Thanks @JC1738

**Pull request merged**